DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<p>Maybe also testing how a color changes with the S/N, because the forced photometry is mainly designed for getting good color measurements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>I meant this exercise to allow users to explore a few other photometric measurements that exist in the catalog, but because of DM suggestions regarding what measurements should be used for colors I left this to focus on gaap. So I have added the suggestion to explore the differences in measurements and S/N when the larger gaap aperture (3.0 instead of 1.0) is used (reportedly because of the larger aperture it decreases S/N).</p>